src/tools/info/ucx_info.c

```diff
@@ -29,6 +29,7 @@ static void usage() {
     printf("                'r' : remote memory access\n");
     printf("                't' : tag matching \n");
     printf("                'w' : wakeup\n");
+    printf("                'e' : error handling\n");
```

Comment:
minor: shouldn't arguments be literal-sorted? (e-error after a-atomic)

Reply:
i wanted to put context options before ep options..